packages/nimble-components/src/rich-text/models/tests/markdown-serializer.spec.ts

```diff
@@ -77,24 +77,87 @@ describe('Markdown serializer', () => {
-                html: '<ul><li><p><em>Bullet list with italics</em></p></li></ul>',
-                markdown: '* *Bullet list with italics*'
+                name: 'Bulleted list with italics',
+                html: '<ul><li><p><em>Bulleted list with italics</em></p></li></ul>',
+                markdown: '* *Bulleted list with italics*'
+            },
+            {
+                name: 'Nested list with levels 1 - Bulleted list, 2 - Numbered list (Bold)',
+                html: '<ul><li><p>Bulleted list</p><ol><li><p><strong>Nested bold numbered list</strong></p></li></ol></li></ul>',
+                markdown:
+                    '* Bulleted list\n\n  1. **Nested bold numbered list**'
+            },
+            {
+                name: 'Nested list with levels 1 - Bulleted list, 2 - Numbered list (Italics)',
+                html: '<ul><li><p>Bulleted list</p><ol><li><p><em>Nested bold numbered list</em></p></li></ol></li></ul>',
+                markdown: '* Bulleted list\n\n  1. *Nested bold numbered list*'
+            },
+            {
+                name: 'Nested list with levels 1- Numbered list (Bold), 2-Bulleted list',
+                html: '<ol><li><p><strong>Numbered list bold</strong></p><ul><li><p>Nested bulleted list</p></li></ul></li></ol>',
+                markdown:
+                    '1. **Numbered list bold**\n\n   * Nested bulleted list'
+            },
+            {
+                name: 'Nested list with levels 1- Numbered list (Italics), 2-Bulleted list',
+                html: '<ol><li><p><em>Numbered list italics</em></p><ul><li><p>Nested bulleted list</p></li></ul></li></ol>',
+                markdown:
+                    '1. *Numbered list italics*\n\n   * Nested bulleted list'
+            },
+            {
+                name: 'Nested list with levels 1- Numbered list, level 2- Bulleted list with multiple items',
+                html: '<ol><li><p>Numbered list</p><ul><li><p>list 1</p></li><li><p>list 2</p></li></ul></li></ol>',
+                markdown: '1. Numbered list\n\n   * list 1\n\n   * list 2'
+            },
+            {
+                name: 'Nested list with levels 1- Bulleted list, level 2- Numbered list with multiple items',
+                html: '<ul><li><p>Bulleted list</p><ol><li><p>list 1</p></li><li><p>list 2</p></li></ol></li></ul>',
+                markdown: '* Bulleted list\n\n  1. list 1\n\n  2. list 2'
+            },
+            {
+                name: 'HTML entities <&>',
+                html: '&',
+                markdown: '&'
+            },
+            {
+                name: 'HTML entities <&trade;>',
+                html: '&trade;',
+                markdown: '™'
+            },
+            {
+                name: 'HTML entities <&euro>',
+                html: '&euro;',
+                markdown: '€'
+            },
+            {
+                name: 'Markdown syntax strings <*>',
+                html: '*',
+                markdown: '\\*'
```

Comment:
Consider using [`String.raw`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/raw) to avoid giving another `\` to escape actual backslash - `\`
```suggestion
                markdown: String.raw`\*`
```